scripts/setup.sh

```diff
@@ -27,7 +27,7 @@ function info()
 info "downloading..."
 curl -O -L https://github.com/c9s/bbgo/releases/download/$version/$dist_file
 tar xzf $dist_file
-mv bbgo-$osf bbgo
+mv bbgo-$osf-amd64 bbgo
```

Comment:
Why adding amd64 here? It could be arm64 because of Apple M1.

Reply:
done